src/main/java/ua/cv/tim/dao/hibernate/AttackArchiveDaoImpl.java

```diff
@@ -0,0 +1,34 @@
+package ua.cv.tim.dao.hibernate;
+
+import org.hibernate.Session;
+import org.hibernate.query.Query;
+import org.springframework.stereotype.Repository;
+import ua.cv.tim.dao.AbstractCrudDao;
+import ua.cv.tim.dao.AttackArchiveDao;
+import ua.cv.tim.model.AttackArchive;
+
+import java.util.List;
+
+/**
+ * Created by rmochetc on 13.02.2017.
+ */
+
+@Repository("AttackArchiveDao")
+public class AttackArchiveDaoImpl extends AbstractCrudDao<AttackArchive> implements AttackArchiveDao {
+
+    @Override
+    public List<AttackArchive> getAll() {
+        Session session = getCurrentSession();
+        Query query = session.createQuery("select aa from AttackArchive aa");
```

Comment:
create session variable is redundant.
use getCurrentSession().createQuery(...) instead